### ISSUE TITLE
Remove `comments: false`

### ISSUE THREAD

**Issue by kentcdodds** [timarney/react-setup#5]
I believe that this goes against most licenses which require licensing information be distributed with all copies of the code. By default `uglify` will include license comments only and remove everything else. It's normally not that huge of a win to remove them.

**Comment by timarney** [comment]
@kentcdodds for webpack 2  do you know if that is done through the loader options plugin config or using the env from the cli 

```
new webpack.LoaderOptionsPlugin({
    test: /\.css$/, // optionally pass test, include and exclude, default affects all loaders
    minimize: true,
    debug: false,
    options: {
        // pass stuff to the loader
    }
})
```

https://github.com/mishoo/UglifyJS2/blob/master/README.md

**Comment by kentcdodds** [comment]
I believe that this is still the same as before (did some testing on my own). I'll file a PR.

**Comment by kentcdodds** [comment]
Ah, just  noticed in the webpack branch that you're using the `-p` flag instead of manually specifying the plugin. You should know that the `-p` flag in webpack 2 is [a shortcut](https://github.com/webpack/webpack/blob/1fbb88818532e9766440afb7af9e2d7a533c3f03/bin/config-optimist.js#L47) for `--optimize-minimize` which actually [includes](https://github.com/webpack/webpack/blob/1fbb88818532e9766440afb7af9e2d7a533c3f03/bin/convert-argv.js#L423-L431) the `LoaderOptionsPlugin`.

If I understand correctly, this means that you have two instances of `LoaderOptionsPlugin` in your config which is probably not what you want. I think it's best to choose either `-p` or specify your plugins yourself (not both). Furthermore, I generally prefer to specify my plugins myself so I can configure Uglify to silence warnings :-)

**Comment by timarney** [comment]
@kentcdodds I played around trying some different settings.

I tried removing the-p option to use and using the UglifyJsPlugin plugin in the webpack 2 config.  I ended up around 211kb vs 141kb with the -p flag.

I also tried setting to production and passing the --optimize-minimize --define process.env.NODE_ENV=\"production\" i.e.  
https://github.com/webpack/webpack/blob/master/bin/config-optimist.js#L47 but I still ended up around 207kb
